scapy/contrib/gtp.py

```diff
@@ -192,14 +192,15 @@ def answers(self, other):
         return (isinstance(other, GTPHeader) and
                 self.version == other.version and
                 self.payload.answers(other.payload))
-
-    @classmethod
-    def dispatch_hook(cls, _pkt=None, *args, **kargs):
-        if _pkt and len(_pkt) >= 1:
-            if (struct.unpack("B", _pkt[0])[0] >> 5) & 0x7 == 2:
-                import gtp_v2
-                return gtp_v2.GTPHeader
-        return GTPHeader
+    

```

Comment:
If you are not changing the code here, please do not mess up with whitespaces as it breaks code history for no reason.